doc/source/user/configoptions.rst

```diff
@@ -142,6 +142,496 @@ Example:
     \endverbatim
 
 
+
+Global configuration options
+----------------------------
+
+Logging
+^^^^^^^
+.. config:: CPL_CURL_VERBOSE
+   :choices: YES, NO
+
+   Set to "YES" to get the curl library to display a lot of verbose information
+   about its operations. Very useful for libcurl and/or protocol debugging and
+   understanding.
+
+.. config:: CPL_DEBUG
+   :choices: ON, OFF, <PREFIX>
+
+   This may be set to ON, OFF or specific prefixes. If it is ON, all debug
+   messages are reported to stdout. If it is OFF or unset no debug messages are
+   reported. If it is set to a particular value, then only debug messages with
+   that "type" value will be reported. For instance debug messages from the HFA
+   driver are normally reported with type "HFA" (seen in the message).
+
+   At the commandline this can also be set with --debug <value> as well as with
+   --config CPL_DEBUG <value>.
+
+.. config:: CPL_LOG
+   :choices: <path>
+
+   This is used for setting the log file path.
+
+.. config:: CPL_LOG_ERRORS
+   :choices: ON, OFF
+
+   Set to "ON" for printing error messages. Use together with "CPL_LOG" for
+   directing them into a file.
+
+.. config:: CPL_TIMESTAMP
+   :choices: ON, OFF
+
+   Set to "ON" to add timestamps to CPL debug messages (so assumes that
+   CPL_DEBUG is enabled)
+
+.. config:: CPL_MAX_ERROR_REPORTS
+
+.. config:: CPL_ACCUM_ERROR_MSG
+
+.. config:: CPL_TMPDIR
+   :choices: <dirname>
+
+   By default, temporary files are written into current working directory.
+   Sometimes this is not optimal and it would be better to write temporary files
+   on bigger or faster drives (SSD).
+
+
+.. config:: CPL_VSIL_CURL_ALLOWED_EXTENSIONS
```

Comment:
Doesn't fit in a "Logging" section (same for CPL_VSIL_CURL_NON_CACHED). Perhaps moved to the "HTTP section" which could be renamed "Networking"  ?